build.gradle.kts

```diff
@@ -407,6 +410,19 @@ project(":grazie") {
     }
 }
 
+project(":js") {
```

Comment:
Don't forget to add the corresponding description in [`CONTRIBUTING.md`](https://github.com/intellij-rust/intellij-rust/blob/master/CONTRIBUTING.md#project-structure)

Reply:
Is such description OK?
```
* `:js` - interop with JavaScript language
```

Reply:
👍